<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabPalette</class>
 <widget class="QWidget" name="TabPalette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="__state_pal_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>State Palette - Right click to copy a single color to the Global palette</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PaletteSwatch" name="state_swatch" native="true">
     <property name="editable" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="copy_palette_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Copy All to Global</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icons/chevron-down.svg</normaloff>:/icons/chevron-down.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="__global_pal_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Global Palette - Left click a color to edit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PaletteSwatch" name="global_swatch" native="true">
     <property name="editable" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaletteSwatch</class>
   <extends>QWidget</extends>
   <header>..ui.paletteswatch.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
